Description: Now, link to system lib: libgatbcore
 <long description that can span multiple lines, optional>
Author: <name and email of author, optional>
Origin: <upstream|backport|vendor|other>, <URL, required except if Author is present>
Bug: <URL to the upstream bug report if any, implies patch has been forwarded, optional>
Bug-<Vendor>: <URL to the vendor bug report if any, optional>
Forwarded: <URL|no|not-needed, useless if you have a Bug field, optional>
Applied-Upstream: <version|URL|commit, identifies patches merged upstream, optional>
Reviewed-by: <name and email of a reviewer, optional>
Last-Update: <YYYY-MM-DD, last update of the meta-information, optional>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -31,7 +31,7 @@
 ################################################################################
 # Define cmake modules directory
 ################################################################################
-SET (GATB_CORE_HOME  ${PROJECT_SOURCE_DIR}/thirdparty/gatb-core/gatb-core)
+SET (GATB_CORE_HOME  /media/DATA/Projets/gatb/gatb-core/gatb-core)
 SET (CMAKE_MODULE_PATH ${GATB_CORE_HOME}/cmake)
 
 ################################################################################
@@ -45,34 +45,30 @@
 # THIRD PARTIES
 ################################################################################
 
-# We don't want to install some GATB-CORE artifacts
-SET (GATB_CORE_EXCLUDE_TOOLS     1)
-SET (GATB_CORE_EXCLUDE_TESTS     1)
-SET (GATB_CORE_EXCLUDE_EXAMPLES  1)
-
 # GATB CORE
-include (GatbCore)
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/")
+find_package(GatbCore 1.3 REQUIRED)
 
 ################################################################################
 # TOOLS
 ################################################################################
 
-# We add the compilation options for the library
-add_definitions (${gatb-core-flags})
+add_definitions(-DSTR_LIBRARY_VERSION="${gatbcore_VERSION}")
 
+# We add the compilation options for the library
+add_definitions(${GATBCORE_DEFINITIONS})
 # We add the gatb-core include directory
-include_directories (${gatb-core-includes})
+include_directories(${GATBCORE_INCLUDE_DIRS})
 
-# We add the path for extra libraries
-link_directories (${gatb-core-extra-libraries-path})
-
-set (PROGRAM_SOURCE_DIR ${PROJECT_SOURCE_DIR}/src)
+set(PROGRAM_SOURCE_DIR ${PROJECT_SOURCE_DIR}/src)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 
-include_directories (${PROGRAM_SOURCE_DIR})
+include_directories(${PROGRAM_SOURCE_DIR})
 file (GLOB_RECURSE  ProjectFiles  ${PROGRAM_SOURCE_DIR}/*.cpp)
 add_executable(${PROJECT_NAME} ${ProjectFiles})
-target_link_libraries(${PROJECT_NAME} ${gatb-core-libraries})
+
+set(GATBCORE_LIBRARIES ${GATBCORE_LIBRARIES} pthread)
+target_link_libraries(${PROJECT_NAME} ${GATBCORE_LIBRARIES})
 
 
 ################################################################################
@@ -85,14 +81,14 @@
 SET (CPACK_PACKAGE_VERSION_MAJOR        "${gatb-tool_VERSION_MAJOR}")
 SET (CPACK_PACKAGE_VERSION_MINOR        "${gatb-tool_VERSION_MINOR}")
 SET (CPACK_PACKAGE_VERSION_PATCH        "${gatb-tool_VERSION_PATCH}")
-SET (CPACK_PACKAGE_VERSION              "${gatb-tool-version}")  
+SET (CPACK_PACKAGE_VERSION              "${gatb-tool-version}")
 
 # We set the kind of archive
 SET (CPACK_GENERATOR                    "TGZ")
 SET (CPACK_SOURCE_GENERATOR             "TGZ")
 
 # We ignore unwated files for the source archive
-SET (CPACK_SOURCE_IGNORE_FILES          
+SET (CPACK_SOURCE_IGNORE_FILES
     "^${PROJECT_SOURCE_DIR}/\\.git/"     ;
     "^${PROJECT_SOURCE_DIR}/\\.gitmodules"  ;
     "^${PROJECT_SOURCE_DIR}/\\.gitignore";
